ahsc_grant/README.md

```diff
@@ -1,47 +1,92 @@
-# AHSC Ridership Prediction Methodology
+# AHSC Ridership Prediction Dashboard
 
-## Overview
+The goal of this tool is to provide predictions to estimate increases in annual bus ridership that correspond to planned increases in scheduled bus services. We use General Transit Feed Specification (GTFS) schedule data, American Community Survey (ACS) data and Ridership data from selected agencies to feed our statistical model. The dashboard summarizes the additional annual riders across each stop that a route goes by. This tool is available to assist mainly AHSC, CARB grant applicants along with other agencies. 
```

Comment:
I think this looks correct! Please add a sentence about how this tool was developed in Fall 2022 for AHSC Round 7 and has not yet been updated.